src/locale/nb.js

```diff
@@ -2,54 +2,54 @@
 //! locale : Norwegian Bokmål [nb]
 //! authors : Espen Hovlandsdal : https://github.com/rexxars
 //!           Sigurd Gartmann : https://github.com/sigurdga
+//!           Ronny Hanssen : https://github.com/spiralis
 
 import moment from '../moment';
 
 export default moment.defineLocale('nb', {
-    months : 'januar_februar_mars_april_mai_juni_juli_august_september_oktober_november_desember'.split('_'),
-    monthsShort : 'jan._feb._mars_april_mai_juni_juli_aug._sep._okt._nov._des.'.split('_'),
-    monthsParseExact : true,
```

Comment:
I don't know what monthsParseExact is meant to do, so it is probably fine to remove it.

Reply:
Not sure neither, and the only examples I could find where really advanced and none of them seemed to apply to Norwegian.

Unless someone else could shed some light on this?